cypress/integration/lbry.spec.js

```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+
+import { getIframeBody } from '../support/commands';
+
+context('<Lbry />', () => {
+  it('it loads Lbry player widget', () => {
+    cy.visit('/iframe.html?id=components-lbry--usage&viewMode=story');
+    cy.get('[data-testId="lbry"]').should('not.be.undefined');
```

Comment:
Remove `data-tesid` assertion

The unit test will check our component renders by finding the `data-test-id` 
The integration test is to find provider specific DOME nodes in the browser.

Reply:
Does this test even pass? The Testing Library data attribute is lowercase "i" on id => ` data-testid="lbry"`